pkg/hosts/hosts.go

```diff
@@ -66,6 +66,7 @@ func (h *Hosts) Add(ip string, hosts []string) error {
 	if err := h.File.Add(ip, hostEntries...); err != nil {
 		return err
 	}
+	h.File.Clean()
```

Comment:
This could at least be wrapped in 
```
if HostsPerLine != -1 {
    h.File.Clean()
}
```
to minimize the platforms where we modify the hosts file.

Reply:
@cfergeau I just used `clean` for windows so it will not affect linux/mac. 

> Short term, we'll have to go with Clean, or use our own fork of hostsfile with https://github.com/cfergeau/hostsfile/commits/splitlines

We just wait for this release otherwise we just fork hostsfile and use it with admin-helper.